kore/test/Test/Kore/ASTVerifier/DefinitionVerifier/Imports.hs

```diff
@@ -752,18 +753,6 @@ aliasVisibilityTests =
         (DeclaringSentence aliasDeclaration)
         (UsingSentence aliasReferenceInNextPatternSentence)
         (SupportingSentences defaultAliasSupportSentences)
-    , nameReferenceTests
```

Comment:
Why is this test removed?

Reply:
I removed this before I figured out the correct implementation for the verification, and it looked like it would never pass under the new rules. I added it back now.